### PR TITLE
Go mod support

### PR DESCRIPTION
**Pull request by asahasrabuddhe**
Added a go.mod file 

Henceforth, we can support go modules through our version tags. Better vendoring :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module github.com/urfave/cli
+
+go 1.12
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	gopkg.in/urfave/cli.v1 v1.20.0
```
Comment (AudriusButkevicius) [inline go.mod]: This is back

Comment (asahasrabuddhe) [reply go.mod]: I think that is there due to the package being referenced in the flag type in the `altsrc` package. We would need to find a way to eventually merge it into the main package and get rid of this too in the process.

**Comment by AudriusButkevicius** [comment]
See #831

**Comment by asahasrabuddhe** [comment]
thanks i will use that instead

**Pull request closed without merging**